coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 87.7,
+  "coverage_score": 90,
   "exclude_path": "",
   "crate_features": "backend-stdio"
```

Comment:
Shouldn't this be `virtio-blk/backend-stdio`?

Reply:
Yes. I'll change it immediately